schema-loader/build.gradle

```diff
@@ -59,3 +59,15 @@ spotbugsMain.reports {
 spotbugsTest.reports {
     html.enabled = true
 }
+
+// for release
+distZip {
+    archiveFileName = "${archivesBaseName}-${project.version}.zip"
+}
```

Comment:
Do we really need this? Are we going to provide a zip archive for users? Sorry I missed it in the previous reviews.

Reply:
I removed it [commit](https://github.com/scalar-labs/scalardb/pull/366/commits/219473b5710f1f7154ae5075eaa1531d48b854d6). Sorry it was my mistake when copying